docs/modules/ROOT/pages/reference/extensions/bean-validator.adoc

```diff
@@ -38,3 +38,28 @@ Or add the coordinates to your existing project:
 ----
 
 Check the xref:user-guide/index.adoc[User guide] for more information about writing Camel Quarkus applications.
+
+== Usage
+
+=== Configuring the ValidatorFactory
+
+Implementation of this extension leverages https://quarkus.io/guides/validation[Quarkus extension].
```

Comment:
```suggestion
Implementation of this extension leverages the https://quarkus.io/guides/validation[Quarkus Hibernate Validator extension].
```

Reply:
Thanks for the corrections. I fixed them all.